modules/scripting-jruby/pom.xml

```diff
@@ -136,22 +136,22 @@
         <dependency>
             <groupId>org.yaml</groupId>
             <artifactId>snakeyaml</artifactId>
-            <version>1.31</version>
+            <version>2.0</version>
         </dependency>
         <dependency>
             <groupId>org.jruby.joni</groupId>
```

Comment:
why are these updates needed?

Reply:
We are updating jrubyversion, hence updating , I think this will also update what snakeyaml  we bundle in Mule 3

Reply:
Moreover snakeyaml 1.31 is vulneranle , we are switching to 2.0